launcher/ui/pages/instance/ScreenshotsPage.cpp

```diff
@@ -325,7 +325,7 @@ void ScreenshotsPage::onItemActivated(QModelIndex index)
         return;
     auto info = m_model->fileInfo(index);
     QString fileName = info.absoluteFilePath();
-    DesktopServices::openFile(info.absoluteFilePath());
+    DesktopServices::openPath(info.absoluteFilePath());
```

Comment:
```suggestion
    DesktopServices::openPath(info);
```
also, delete the fileName variable as it's completely unused.

Reply:
I love bad code!!!